handlers/labels.go

```diff
@@ -0,0 +1,55 @@
+package handlers
+
+import (
+	"log"
+	"strconv"
+)
+
+const (
+	// initialReplicasCount how many replicas to start of creating for a function, this is
+	// also used as the default return value for getMinReplicaCount
+	initialReplicasCount = 1
+
+	// OFFunctionNameLabel is the label key used by OpenFaaS to store the function name
+	// on the resources managed by OpenFaaS for that function.  This key is also used to
+	// denote that a resource is a "Function"
+	OFFunctionNameLabel = "faas_function"
```

Comment:
What does the `OF` prefix add?

Reply:
In retrospect, those aren't needed.  I will remove them